### ISSUE TITLE
JSDoc comments not shown in signature help for instance properties

### ISSUE THREAD

**Issue by ManuKle** [Microsoft/TypeScript#20894]
**TypeScript Version:**  2.7.0-dev.20171224

**Code**

```ts
export class Class {
  /** I'm not always showing up */
  method: () => void = () => console.log("Hello, I'm a bug")
}

const test = new Class();
test.method();

```

**Expected behavior:**
The description should be shown like after pressing "."
![image](https://user-images.githubusercontent.com/31354932/34342472-82fbca6e-e9b1-11e7-8cf6-67ac9d023439.png)

or like here
![image](https://user-images.githubusercontent.com/31354932/34342492-e94cbb8e-e9b1-11e7-984b-af76604f8f84.png)

**Actual behavior:**
The description is hidden once the brackets are inserted
![image](https://user-images.githubusercontent.com/31354932/34342499-1d4242e2-e9b2-11e7-9282-b758f28d9f0b.png)

**Comment by DanielRosenwasser** [comment]
Likely related to #19231.

**Comment by andy-ms** [comment]
Fixed by #21907
